Lib/test/test_asyncio/test_tasks.py

```diff
@@ -4,6 +4,7 @@
 import contextlib
 import functools
 import io
+import gc
```

Comment:
Another nit: 'gc' import should be before 'functools' -- we sort imports alphabetically in CPython.